examples/Modals/ModalFooter.md

```diff
@@ -0,0 +1,40 @@
+This is a utility component for adding a footer row to the `<Modal>`. It's set up to use `flexbox`, and fill in buttons from right to left. For example:
```

Comment:
Should this be 'left to right'? If my comment from line 14 is wrong, you can disregard this

Reply:
rationale for "right to left" above